containers/podman-desktop
# feat: add install system-wide button for compose onboarding

feat: add install system-wide preference button for compose extension

### What does this PR do?

* Add a "toggle" button for installing compose system wide
* Adds the first initial compose onboarding sequence which will only appear when compose isn't installed system wide (for now, as that's the only functionality at the moment / setting).
* Button will disappear / after it has been installed system wide.

### Screenshot/screencast of this PR




https://github.com/containers/podman-desktop/assets/6422176/3535f0b1-48ae-49c9-b5d0-696e93389f5f


### What issues does this PR fix or reference?




### How to test this PR?



1. Type `whereis docker-compose` and delete it (most likely
   /usr/local/bin/docker-compose.
2. Make sure you have the binary still in
   .local/share/containers/podman-desktop
3. Click on onboarding for Compose
4. Press the toggle button for system-wide install.
